tests/test_pypuck.py

```diff
@@ -1 +1,65 @@
+# authors: Jarvis Nederlof
+# date: 2020-03-02
+
+"""
+This script tests the pypuck functions in the pypuck module.
+"""
+
 from pypuck import pypuck
+import pandas as pd
+
+# @pytest.mark.skip()
+def test_player_stats_good(start_date='2019-10-02', end_date='2020-02-28'):
+	"""
+	Test function to check proper inputs and returns.
+
+	Keyword Arguments:
+		start_date {str} -- start_date to query (default: {'2019-10-02'})
+		end_date {str} -- end_date to query (default: {'2020-02-28'})
+
+	Raises:
+		TypeError: A message if return type is wrong (wrong data type).
+		ValueError: A message if return value is wrong (not enough data).
+	"""
+	# Test for various end_dates
+	for end_date in [end_date, None]:
+		df = pypuck.player_stats(start_date, end_date)
```

Comment:
```suggestion
	for end_date_run in [end_date, None]:
		df = pypuck.player_stats(start_date, end_date_run)
```
In your original code does end_date gets overwriten? And we'll end up with end_date = None? I guess in the test case it's alright, but in general it might be a confusing choice of variable names.

Reply:
I actually purposely chose to overwrite it because then it would be None for the test on the next line(s) for different start date variables. Thus testing more values.

Reply:
Ok, got it! Thank you!